### ISSUE TITLE
Unlickable links in RSS feeds since Dec 28, 2022 push to master tree

### ISSUE THREAD

**Issue by Antorell** [Novik/ruTorrent#2426]
### Please complete the following tasks.

- [X] Web browser cache cleared
- [X] Link provided to install script if applicable
- [X] Web browser, ruTorrent, PHP and OS version provided

### Tell us about your environment

ruTorrent v4.0 beta (master Dec 28, 2022)
php 8.1
Ubuntu 22.10

### Tell us how you installed ruTorrent

git clone from the master branch (Dec 28, 2022)

### Describe the bug

Posting a bug report before I forget. 

There's still an issue with the new RSS changes that were pushed to the master branch the Dec 28, 2022. The links in the description of the feeds are not clickable anymore. The links are colored like other links, or the links before the RSS changes, but they cannot be clicked. Feeds can be loaded without problems.

Happy new year. 

### Steps to reproduce

Load a RSS feed that has links to the release page or external links

Feed with links: https://feed.animetosho.org/rss2

### Additional context

![Screenshot (52)](https://user-images.githubusercontent.com/25066224/210150900-b0b22e04-0218-4b41-a8c6-279cc4947e88.png)

**Comment by stickz** [comment]
Hi @Antorell, this is intentional for security reasons. It could cause serious phishing implementations, if we don't disable links.

**Comment by Antorell** [comment]
Yes, the other day I didn't check the sanitize.config.js in the js/ folder and thought the RESTRICTED keyword at the end of sanitize.config in the init.js of the RSS plugin was fixed feature and not an option. 

I switched RESTRICTED to BASIC in the init.js of the RSS plugin, and it works like before the sanitize changes. I also added a _blank parameter to add attribute so links open in a new tab.

```
   add_attributes: {
     'a': {
            'rel': 'nofollow',
	    'target': '_blank'   
          }
   },
```

Cheers
